### DISCUSSION TITLE
Data Product with configuration state or without

### DISCUSSION THREAD

**Discussion by FroehlichMarcel** [EKGF/dprod#18] [General]
Following the discussion last week, I propose to
push the decision to capture configuraton state or not to the ontology user
by introducing a separate (sub?)class dp:Release, that can be used to 
keep such state.

**Comment by joshcornejo** [comment]
Isn't state in the case of an Asset better represented as a property?

I agree with specialisation when the change encompasses significant semantic differences (for example a 'template' `odrl:Policy` is an `odrl:Offer` when a supplier is "offering" to the market, and then each contract turns to an `odrl:Agreement`). 

Whilst in DCAT/DPROD, the asset is always a "singleton" (endpoint or group of endpoints) and is either "draft" or "release"?
